atom/browser/api/atom_api_app.cc

```diff
@@ -830,6 +831,17 @@ void App::SetAppPath(const base::FilePath& app_path) {
   app_path_ = app_path;
 }
 
+#if !defined(OS_MACOSX)
+void App::SetAppLogsPath() {
```

Comment:
Wouldn't it be better to allow users provide a custom logs path as an argument rather than expecting them to change it with `app.setPath` ?

Reply:
@deepak1556 what about if we let this be the default and let them pass an argument optionally?

Reply:
Yup I am good with that,

Reply:
done!